test/test_client.py

```diff
@@ -1686,7 +1688,71 @@ def test_srv_max_hosts_kwarg(self):
             srvmaxhosts=2)
         self.assertEqual(
             len(client.topology_description.server_descriptions()), 2)
-
+    
+    @client_context.require_auth
+    @client_context.require_version_min(4, 7, -1)
+    def test_helpers_with_comment(self):
```

Comment:
Let's refactor these tests out of test_client/test_database/test_collection and into a single test_comment.py file. Then you can reuse this test for all helpers instead of duplicating. See test_session for inspiration, it also tests passing a parameter ("session") to every api method.

Reply:
Done.